src/app/auth/settings/account-settings.component.ts

```diff
@@ -23,10 +25,17 @@ export class AccountSettingsComponent extends BaseComponent implements OnInit {
 
     public activeTab: AccountSettingsTabs = AccountSettingsTabs.PROFILE;
     public user$: Observable<MaybeNull<AccountFragment>>;
+    public adminPriveleges$: Observable<boolean>;
 
     private router = inject(Router);
     private route = inject(ActivatedRoute);
     private loggedUserService = inject(LoggedUserService);
+    private localStorageService = inject(LocalStorageService);
+
+    public componentData$: Observable<{
+        user: MaybeNull<AccountFragment>;
+        adminPriveleges: boolean;
+    }>;
```

Comment:
This name tells nothing. 
Also, there cannot be privileges if there is no user.
I suggest a struct which contains these fields, and it may be null as a whole.

Reply:
Renamed and fixed